pkg/domain/infra/abi/play.go

```diff
@@ -359,6 +414,11 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 	containerConfig.Image = containerYAML.Image
 	containerConfig.ImageID = newImage.ID()
 	containerConfig.Name = containerYAML.Name
+
+	if podName != "" {
+		containerConfig.Name = fmt.Sprintf("%s-%s", podName, containerYAML.Name)
+	}
+
```

Comment:
We've some examples in the man page that probably ought to be touched up based on these changes.

Reply:
Will do

Reply:
@TomSweeneyRedHat I looked at [this](https://github.com/containers/libpod/blob/master/docs/source/markdown/podman-play-kube.1.md) which seemed like what you were talking about, but there doesn't seem to be anything related to the name of the containers in there which might need changing.

Should I add something about this in there ?

Reply:
@theunrealgeek my bad, I thought the output in those examples were getting a dash added to them.  Ignore this.